mmdet/ops/dcn/functions/modulated_dcn_func.py

```diff
@@ -0,0 +1,158 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import torch
+from torch.autograd import Function
+
+from .. import modulated_dcn_cuda as _backend
+
+
+class ModulatedDeformConvFunction(Function):
+
+    def __init__(ctx, stride, padding, dilation=1, deformable_groups=1):
+        super(ModulatedDeformConvFunction, ctx).__init__()
+        ctx.stride = stride
+        ctx.padding = padding
+        ctx.dilation = dilation
+        ctx.deformable_groups = deformable_groups
```

Comment:
The `__init__` method is not necessary. Arguments like `stride`, `padding` can be set in the forward method.